### PR TITLE
Update RU translation in admin

### PR DESCRIPTION
**Pull request by vlad320**
#### Description of what you did:

Enhancement Russian translation.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by vlad320** [comment]
Hey.  
I'm sorry. This is my first pull request in my life :)

If approved, can I delete the branch on my fork? Or should I merge it with the master also at fork?

File: packages/strapi-plugin-users-permissions/admin/src/translations/ru.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "Auth.form.error.confirmed": "Адрес электронной почты не подтвержден.",
   "Auth.form.error.email.invalid": "Неправильный адрес электронной почты.",
   "Auth.form.error.email.provide": "Укажите свое имя пользователя или адрес электронной почты.",
-  "Auth.form.error.email.taken": "Почтовый адрес уже используется",
+  "Auth.form.error.email.taken": "Данный Email уже используется",
```
Comment (chiliec) [inline packages/strapi-plugin-users-permissions/admin/src/translations/ru.json]: Why everywhere "email" with small letter, but there with a capital?

Comment (vlad320) [reply packages/strapi-plugin-users-permissions/admin/src/translations/ru.json]: @chiliec 
Hello, this is my fault. I’ll update now PR

**Comment by chiliec** [comment]
@soupette hi, can you merge it? It's good translation update :)

**Pull request merged**